### PR TITLE
[REF-5813] mcenter-cli: ee fixes

### PR DESCRIPTION
**Pull request by liororama**
o adding upload download list
o adding ability to attach mlapp to an existing ee while uploading
o small fix to the get_l3_v2 api call to get start/end parameters

### CODE REVIEW

File: reflex-common/reflex-common/src/main/python/mcenter_cli/parallelm/mcenter_cli/execution_environment.py [hunk 1]
```diff
@@ -19,6 +19,17 @@ def del_if_exists(d, field):
         del(d[field])
 
 
+def ee_get_json(mclient, ee_name):
+    all_ees = mclient.ee_list()
+    ee = list(filter(lambda x: x["name"] == ee_name, all_ees))
```
Comment (zohar-mizrahi) [inline reflex-common/reflex-common/src/main/python/mcenter_cli/parallelm/mcenter_cli/execution_environment.py:24]: Two lightning comment :) 
1. As a pythonist, I'd always encourage to use list comprehensions. IMHO, it is more clear.
2. In terms of performance, in cases where you provide a lambda, it is usually better to use list comprehension. The 'map', 'filter', etc, are better when you already have a defined function.

**Review by zohar-mizrahi — APPROVED** [review]

File: reflex-common/reflex-common/src/main/python/mcenter_cli/parallelm/mcenter_cli/execution_environment.py [hunk 2]
```diff
@@ -19,6 +19,17 @@ def del_if_exists(d, field):
...
+    if len(ee) > 1:
+        raise Exception("More then one EE with the same name!!!!")
```
Comment (yakov-g) [inline reflex-common/reflex-common/src/main/python/mcenter_cli/parallelm/mcenter_cli/execution_environment.py:26]: I think it should be "than"
Aslo, as we are facing customers (or open source), let's discuss warning messages guidelines. I think one ! is enough ;)

**Review by yakov-g — APPROVED** [review]

File: reflex-common/reflex-common/src/main/python/mcenter_cli/parallelm/mcenter_cli/execution_environment.py (hunk 2 above)
Comment (zohar-mizrahi) [reply reflex-common/reflex-common/src/main/python/mcenter_cli/parallelm/mcenter_cli/execution_environment.py:26]: I agree with Yakov

File: reflex-common/reflex-common/src/main/python/mcenter_cli/parallelm/mcenter_cli/execution_environment.py (hunk 1 above)
Comment (liororama) [reply reflex-common/reflex-common/src/main/python/mcenter_cli/parallelm/mcenter_cli/execution_environment.py:24]: Fixed

**Pull request merged**